src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Text.Json
     public ref partial struct Utf8JsonReader
     {
         /// <summary>
-        /// Reads the next JSON token value from the source, unescaped, and transcoded as a <see cref="string"/>.
+        /// Parses the current JSON token value unescaped and transcoded as a <see cref="string"/>.
```

Comment:
nit: Same here/elsewhere, I would keep `from the source`.

Also nit: Imo, keeping the comma helps with clarity while reading it.